gstd/src/sync/mutex.rs

```diff
@@ -121,6 +140,15 @@ impl<T> Mutex<T> {
     fn locked_by_mut(&self) -> &mut Option<(MessageId, BlockNumber)> {
         unsafe { &mut *self.locked.get() }
     }
+
+    fn get_or_set_id(&self) -> MutexId {
```

Comment:
so is it getter or setter tho?

Reply:
It is sort of lazy assignment. Renamed to make it more clear